### ISSUE TITLE
https://huggingface.co/spaces/chiliktam/OpenVoiceV2 non existent, can't build

### ISSUE THREAD

**Issue by Carlos-not-found** [maximofn/openVoiceV2_docker#1]
The page described in the Dockerfile doesn't exist, because of this, trying to build will not work.

**Comment by maximofn** [comment]
That's right, that's because the creator of the HuggingFace space has deleted it. That's why I created the image. 

What you can do is download the docker hub image like this

``` bash
docker pull maximofn/open_voice_v2:latest
```

And run it like this

``` bash
./run_app.sh
```

If hopefully the models are still in huggingface you can have the space in docker

**Comment by factor3** [comment]
Small problem:

Doing the docker pull that you describe doesn't work either. Doing a 

` $ docker pull maximofn/open_voice_v2:latest`

yields the following:

`Error response from daemon: pull access denied for maximofn/open_voice_v2, repository does not exist or may require 'docker login': denied: requested access to the resource is denied`

So there is no way to install this container.

I would like to note that the Huggingface models appear to still be available.

**Comment by maximofn** [comment]
Yes, I don't know why it was deleted. This weekend, when I have some time I'll upload it again.

**Comment by maximofn** [comment]
I just uploaded the image to docker hub, so if you do docker pull maximofn/open_voice_v2:1.0.1 it should be downloaded

**Comment by factor3** [comment]
Thank you. 

Suggestion: you might want to change the documentation so that it suggests downloading maximofn/open_voice_v2:1.0.1 instead of maximofn/open_voice_v2:latest

**Comment by maximofn** [comment]
Sure
